starter-code/pizza.js

```diff
@@ -1 +1,81 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function(){
+
+$('.panel strong').text("$"+13)
+
+var greenPepper = $(".green-pepper");
+var btnGP = $(".btn-green-peppers");
+
+var pep = $('.pep');
+var btnPep = $(".btn-pepperonni");
+
+var mushroom = $('.mushroom');
+var btnMush = $('.btn-mushrooms');
+
+var whiteSauce = $('.white-sauce');
+var sauce = $('.sauce');
+var btnSauce = $(".btn-sauce");
+
+var glutenFreeCrust = $('.crust-gluten-free');
+var crust = $('.crust');
+var btnCrust = $('.btn-crust')
+
+var priceList = $('.price ul li');
+
+$('*').click(function() {
+  var total = 10;
+  for(var i=0; i < priceList.length; i++){
+    var oneItem = priceList.get(i);
+    if ($(oneItem).is(":visible")) {
+    resultStr = oneItem.innerHTML;
+    resultStr = parseFloat(resultStr.slice(1));
+    total += resultStr;
+    }
+    $('strong').text("$" + total)
+  }
+});
+
+var removeToppings = function(btn, top, str) {
+  btn.click(function() {
+  top.toggle();
```

Comment:
This is good practice!
It might have seemed a bit overkill for a five-button interface, but on bigger scales this is definitely how you want to proceed to keep your code DRY (as in Don't Repeat Yourself) :)